### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by iheanyi**

### CODE REVIEW

**Comment by iheanyi** [comment]
foo

**Comment by iheanyi** [comment]
hm

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,8 +19,7 @@ jobs:
     name: "CI test"
     runs-on: ubuntu-16.04
     steps:
-      - run: env
-      - run: "echo $THE_TEST_SECRET"
-      - run: "echo '${{ github.secrets.THE_TEST_SECRET }}' | wc"
+      - run: "echo '${{ secrets.THE_TEST_SECRET }}' | wc"
```
Comment (iheanyi) [inline .github/workflows/ci.yml]: This is a comment

**Comment by andymckay** [comment]
A comment

**Pull request closed without merging**